test/interface/ode_strip_test.jl

```diff
@@ -11,14 +11,30 @@ u0 = [1.0; 0.0; 0.0]
 tspan = (0.0, 0.5)
 prob = ODEProblem(lorenz!, u0, tspan)
 
-sol = solve(prob, Rosenbrock23())
+rosenbrock_sol = solve(prob, Rosenbrock23())
+TRBDF_sol = solve(prob, TRBDF2())
+vern_sol = solve(prob,Vern6())
+@testset "Interpolation Stripping" begin
+    @test isnothing(SciMLBase.strip_interpolation(rosenbrock_sol.interp).f)
+    @test isnothing(SciMLBase.strip_interpolation(rosenbrock_sol.interp).cache.jac_config)
+    @test isnothing(SciMLBase.strip_interpolation(rosenbrock_sol.interp).cache.grad_config)
+end
+
+@testset "Rosenbrock Solution Stripping" begin
+    @test isnothing(SciMLBase.strip_solution(rosenbrock_sol).prob)
+    @test isnothing(SciMLBase.strip_solution(rosenbrock_sol).alg)
+    @test isnothing(SciMLBase.strip_solution(rosenbrock_sol).interp.f)
+    @test isnothing(SciMLBase.strip_solution(rosenbrock_sol).interp.cache.jac_config)
+    @test isnothing(SciMLBase.strip_solution(rosenbrock_sol).interp.cache.grad_config)
+end 
+
+@testset "TRBDF Solution Stripping" begin 
+    @test isnothing(SciMLBase.strip_solution(TRBDF_sol).prob)
+    @test isnothing(SciMLBase.strip_solution(TRBDF_sol).alg)
+    @test isnothing(SciMLBase.strip_solution(TRBDF_sol).interp.f)
+    @test isnothing(SciMLBase.strip_solution(TRBDF_sol).interp.cache.jac_config)
+    @test isnothing(SciMLBase.strip_solution(TRBDF_sol).interp.cache.grad_config)
+end
 
-@test isnothing(SciMLBase.strip_interpolation(sol.interp).f)
-@test isnothing(SciMLBase.strip_interpolation(sol.interp).cache.jac_config)
-@test isnothing(SciMLBase.strip_interpolation(sol.interp).cache.grad_config)
 
-@test isnothing(SciMLBase.strip_solution(sol).prob)
-@test isnothing(SciMLBase.strip_solution(sol).alg)
-@test isnothing(SciMLBase.strip_solution(sol).interp.f)
-@test isnothing(SciMLBase.strip_solution(sol).interp.cache.jac_config)
-@test isnothing(SciMLBase.strip_solution(sol).interp.cache.grad_config)
\ No newline at end of file
+@test_throws ErrorException SciMLBase.strip_solution(vern_sol)
```

Comment:
It shouldn't be a generic error?

Reply:
What kind of error should this be?

Reply:
LazyInterpolationException or something, Make a custom one so the GUI can catch it.